tests/rustdoc-ui/feature-gate-doc_cfg_hide.rs

```diff
@@ -1,5 +1,6 @@
-#![doc(cfg_hide(test))]
-//~^ ERROR `#[doc(cfg_hide)]` is experimental
+// FIXME: Remove this file once feature is removed
```

Comment:
Hasn't it been removed now?

Also, can you add some tests to make sure `doc(auto_cfg)` (perhaps with some different combinations of arguments, including none, to `auto_cfg`) and `doc(cfg)` are still feature-gated?

Reply:
Indeed and sure.